src/js/models/bounds.js

```diff
@@ -165,9 +167,12 @@ define(function (require, exports, module) {
                 case layerLib.layerKinds.TEXT:
                     boundsObject = descriptor.boundingBox;
                     break;
-                default:
+                case layerLib.layerKinds.VECTOR:
                     boundsObject = descriptor.boundsNoEffects;
```

Comment:
this is only for the case that pathBounds does not exist, which i was hitting fairly regularly